cypress/support/commands.js

```diff
@@ -23,3 +23,235 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
```

Comment:
Nonblocking: This single file tends to be lengthy. Better to start breaking it into multiple files for ease of reading and searching. E.g you can have a 

dashboard_commands.js
visualization_commands.js

and then the current file can keep some generic commands.

Reply:
While splitting the Cypress commands across multiple files should be straight-forward, I will wait until after the test environment loading and unloading functionality has been merged to make the change in order to limit of number of merge conflicts for the next PR.